src/component/admin/apiToken/ApiTokenList/ProjectsList/ProjectsList.tsx

```diff
@@ -24,7 +24,7 @@ export const ProjectsList: VFC<IProjectsListProps> = ({
     return (
         <>
             {fields.map((item, index) => (
-                <Fragment key={item}>
+                <Fragment key={item || '*'}>
```

Comment:
There is an edge case, because `key` cannot be `""`. If projects are `[]` (that shouldn't happened, I don't think it's allowed API state), it throws React warning.